src/main/scripts/upload_required_plugins.py

```diff
@@ -0,0 +1,68 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+import json
+import requests
+import jq
+import sys
+
+# Get the Package name and Package version.
+package_name = sys.argv[1]
+package_version = sys.argv[2]
+
+
+# Get the specs of the Plugin from response.
+res = requests.get(f"https://hub.cdap.io/v2/packages/{package_name}/{package_version}/spec.json")
+assert res.ok or print(f"Specs not found for the Plugin. Status code: {res.status_code}")
+response_data = res.json()
+actions = response_data["actions"][0]
+plugin_details = actions["arguments"]
+
+# Get the required details from specs.
+plugin_name = plugin_details[0]["value"]
+plugin_version = plugin_details[1]["value"]
+plugin_config = plugin_details[3]["value"]
+plugin_jar = plugin_details[4]["value"]
```

Comment:
Rely on the `name` fields in the `arguments` array elements, instead of list indices.

Reply:
Updated, used the name fields to extract the specs. [1d4cda1ac469839b331b485353f5032cc5f1a469]